Library/Homebrew/test/tap_test.rb

```diff
@@ -184,10 +175,10 @@ def test_git_variant
     touch @path/"README"
     setup_git_repo
 
-    assert_equal "e1893a6bd191ba895c71b652ff8376a6114c7fa7", @tap.git_head
-    assert_equal "e189", @tap.git_short_head
-    assert_match "years ago", @tap.git_last_commit
```

Comment:
I'll draw particular attention to this test, because it was the only one that needed any modification other than an updated SHA. (I had to remove the "years".)